hushline/settings/__init__.py

```diff
@@ -54,9 +55,19 @@
     UpdateBrandAppNameForm,
     UpdateBrandLogoForm,
     UpdateBrandPrimaryColorForm,
+    UpdateDirectoryIntroTextForm,
 )
 
 
+def sanitize_input(input_text: str) -> str:
```

Comment:
We should add this to the docs that these are the allowed fields/attributes. Possibly also in the UI.